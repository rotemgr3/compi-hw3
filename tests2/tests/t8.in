//piazza 72 - should be errorMainOverride +
override void main() {int a = 5;}