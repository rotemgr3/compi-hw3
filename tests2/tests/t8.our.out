line 2: main is not allowed to be overridden
